---
tags: [gene, biology, algorithm]
---

# RNA Secondary Structure

RNA secondary structure is the result of the [RNA Folding](202310311043.md).
Within the structure there are two main types of substructure that is match
regions, i.e., **stems**, and unpaired regions, i.e., **loops**. Loops can be
further categorised into **internal loops** (connect 2 stems and formed by 2
strands of equal/unequal sizes), **the bulges** (unpaired strands occurring in
only one side of the stems), **terminal loops** (located at the extremity of a
steam and have at least 4 [nucleotides](202308082154.md)), and **multiloops**
(connects 3 or more stems).

If there is high reactivity, the nucleotide is easily accessible, thus is not
paired. If there is low reactivity, then the nucleotide is involved in pairing.

A RNA secondary structure is responsible for RNA folding, RNA stabilisation,
splicing, translation, regulation of [Gene Expression](202308091143.md), and
interaction with other molecules.

The secondary structure prediction could be done with three approaches:
thermodynamic, comparative, and hybrid. Thermodynamic approach find the most
stable structure that has the minimum free energy. However, this is limited by
the uncertainty of the energy model due to the external factors such as other
macromolecules. Examples of this approach are Mfold and RNAfold, most of them
has time complexity of $O(n^3)$. Comparative approach instead utilises multiple
homologous RNA sequences in search for covariation between nucleotides of
different sequences, that is compensatory mutations that maintain the BP and
sequence folding. But this approach is limited by its dependency on the
[Homologous](202308160957.md) sequences used for comparison and quality of
alignment. Examples of this approach are Foldalign and Tfold. RNAalifold is a
renown example of the hybrid approach.

The optimal solution from the prediction is not always the good solution since
it depends on the RNA environment.
