---
tags: [algorithm, biology]
---

# Sequence Alignment

Sequence alignment analyses many [DNA](202308082154.md) and
[protein](202308082207.md) sequences to identify domains or motifs shared among
a group of molecules. It could be improved by implementing a scoring system to
compare different alignments and include penalties to prevent unreasonable
number of insertions. Alignment aids in establishing evolutionary relationship
by constructing a [Phylogenetic Tree](202308142026.md).

Alignment could be done globally or locally. Global alignment forces alignment
to span the entire length of all query sequences in order to find out similarity
between two proteins. Local alignment is often preferable due to the divergent
nature of sequences, but because of the challenge of identifying the regions of
similarity, it could be more difficult to calculate. Thus, local alignment is
best for comparing specific region between the two. There could have more than
one output from local alignment.

Sequence similarity usually implies functional and/or structural relationship
between proteins. It could be used in sequence against database to query
[Sequence Database](202308152116.md) to find matches, against sequence to
establish evolutionary relationship between the two proteins, or against
structure to generate input alignment for comparative modelling or protein
threading.

**Note**: [Homology](202308160957.md) is a qualitative statement specifies about
a common ancestral relationship. It is different from quantitative measurement
of similarity in which the latter calculates the percentage of aligned residues
in physiochemical properties.

- [Pairwise Sequence Alignment](202308172014.md)
- [Multiple Sequence Alignment](202308172015.md)
