---
tags: [graphics, tree, biology]
---

# Phylogenetic Tree

Phylogenetic tree is a branching diagram or a [Tree](202112101956.md) showing
the evolutionary relationships among various biological species or other
entities based upon similarities and differences in their physical or genetic
characteristics. A single node represents a taxonomic unit like species, genes,
or proteins. A branch represents the estimate time of the evolutionary
relationships among the taxonomic units Its branch's length is proportional to
the number of [Amino Acid](202308082207.md) differences between sequences. The
diagram can be depicted as **dendrogram** (scaled, and proportional to the
amount of evolutionary divergence), **cladograms** (unscaled branching
hierarchical tree), and **phylograms** (hierarchical
[cluster](202308212101.md)).

**Note**: Root depicts the common ancestor of the life forms in a **rooted
tree**. Further, the direction indicates the evolutionary time in such a tree.
An unrooted tree can do none of these.

We can find out all possible unrooted trees and rooted trees from
[the number of taxonomy](202308142025.md) $n$. For unrooted trees, calculate
$\frac{2n-5}{(n-3)2^{n-3}}$ where $n>2$. For rooted trees, calculate
$\frac{2n-3}{(n-2)2^{n-2}}$ where $n>2$.

From the tree, we could group nodes into sister groups, that is nodes that split
from the same ancestor, and outgroup, that is nodes that don't have this
feature. If a named taxon's constituent taxa are more closely related to one
another than any taxon outside the clade, then it constitutes a *monophyletic
clade*.

Distance-based data types' value are calculated by measuring the distance
between species or genes. Character-based data types' value are calculated based
on morphological features such as number of legs, hair, wings, scales etc. or
DNA/protein sequences. Character-based data types can be converted into
distance-based data types, but not the other way around.
